UpdateDependencies.ps1

```diff
@@ -0,0 +1,113 @@
+﻿#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# This script updates all the project.json files with the latest CoreFX build version,
+# and then creates a Pull Request for the change.
+
+param(
+    [Parameter(Mandatory=$true)][string]$GitHubUser,
+    [Parameter(Mandatory=$true)][string]$GitHubEmail,
+    [Parameter(Mandatory=$true)][string]$GitHubPassword,
+    [string]$CoreFxVersionUrl='https://raw.githubusercontent.com/dotnet/versions/master/dotnet/corefx/master/Latest.txt', 
```

Comment:
Yes.  This probably won't be the final location, given our conversation last week.  But the idea is that this will be an optional parameter going forward.  It felt wrong hard-coding 'https://github.com/eerhardt/versions' in the corefx repo, so I optimistically picked a URL.

Do you think it would be better to just make this mandatory now, since it doesn't work if you don't supply the actual value?  Or should I hard-code the path to 'eerhardt/versions' for now?  Or leave it as-is?